hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -965,7 +974,7 @@
       uvm_test_seq: chip_sw_keymgr_key_derivation_vseq
       sw_images: ["//sw/device/tests/sim_dv:keymgr_key_derivation_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+lc_at_prod=1", "+sw_test_timeout_ns=20_000_000"]
+      run_opts: ["+use_otp_image=LcStProd", "+sw_test_timeout_ns=20_000_000"]
```

Comment:
@weicaiyang let me know if this is ok.

Reply:
this probably works. I will completed remove the`lc_at_prod` in the sequence after this PR is in. Thanks

Reply:
oh, with this change, this test will probably fail as we need `lc_at_prod` flag to decide what predicted value is.
Perhaps don't touch this one and I will clean it up later

Reply:
Ok sounds good - I will revert it.